pkg/config/config.go

```diff
@@ -594,12 +594,16 @@ type Input struct {
 	Required bool `json:"required,omitempty"`
 }
 
-// The root melange configuration
+// Configuration is the root melange configuration.
 type Configuration struct {
 	// Package metadata
 	Package Package `json:"package" yaml:"package"`
 	// The specification for the packages build environment
 	Environment apko_types.ImageConfiguration `json:"environment" yaml:"environment"`
+	// TODO: move them a runtime specialized section.
+	// Linux process capabilities to add to the pipeline container.
+	CapAdd []string `json:"cap-add,omitempty" yaml:"cap-add,omitempty"`
```

Comment:
Can you introduce a new Capabilities object to track what's in Build? We probably want the ability to "drop" capabilities too and it would be nice if they lived in the same struct and mirrored Build for clarity. I'd also make this just "add" so it'd look something like:

```yaml
capabilities:
  add: ["SETUID"]
```

And in the future:

```yaml
capabilities:
  add: ["SETUID"]
  drop: ["NET_ADMIN"]
```

Or whatever.

Reply:
I like it. I just addressed it, alongside the capability drop option.
Do we want to add a specialized test for this?

Reply:
This is great, yes, can you add this as a test and in the examples/ ?

Reply:
Thanks @jonjohnsonjr Just added e2e tests for both build and test.
Plus a pipeline example with `iproute`.